packages/components/switch/src/switch.vue

```diff
@@ -45,12 +45,18 @@
         <el-icon v-if="loading" :class="ns.is('loading')">
           <loading />
         </el-icon>
-        <el-icon v-else-if="activeActionIcon && checked">
-          <component :is="activeActionIcon" />
-        </el-icon>
-        <el-icon v-else-if="inactiveActionIcon && !checked">
-          <component :is="inactiveActionIcon" />
-        </el-icon>
+        <template v-else-if="checked">
+          <el-icon v-if="activeActionIcon || $slots.activeActionIcon">
+            <component :is="activeActionIcon" v-if="activeActionIcon" />
+            <slot v-else name="activeActionIcon" />
+          </el-icon>
+        </template>
+        <template v-else-if="!checked">
+          <el-icon v-if="inactiveActionIcon || $slots.inactiveActionIcon">
+            <component :is="inactiveActionIcon" v-if="inactiveActionIcon" />
+            <slot v-else name="inactiveActionIcon" />
+          </el-icon>
+        </template>
```

Comment:
```suggestion
        <slot v-else-if="checked" name="activeActionIcon">
          <el-icon v-if="activeActionIcon">
            <component :is="activeActionIcon" />
          </el-icon>
        </slot>
        <slot v-else-if="!checked" name="inactiveActionIcon">
          <el-icon v-if="inactiveActionIcon">
            <component :is="inactiveActionIcon" />
          </el-icon>
        </slot>
```
Sometimes users want to add other things like words or images in slots. @btea What do you think?

Reply:
The code has been modified as suggested